earth_enterprise/rpms/opengee-common/pre-install.sh

```diff
@@ -24,9 +24,6 @@ check_group()
     # Add group if it does not exist:
     if [ -z "$GROUP_EXISTS" ]; then
         groupadd -r "$GEGROUP" &> /dev/null
-        keyvalue_file_set "$GEE_INSTALL_KV_PATH" gegroup_existed "false"
```

Comment:
Old comment: This used to part of determining whether to change ownership and 'upgrade' a Fusion asset root. Now it seems to be simply omitted.  Are we sure that a user can't bring in an old asset root with the wrong group?

Reply:
There is a different way that this is checked for now, and the post-install.sh treats the rpm fresh install as an "upgrade" install for this if there was a prior non-rpm install found.  This should produce the same behavior as before, because gegroup_existed detection was actually used indirectly to determine install/upgrade behavior in post-install.sh prior.

Reply:
So you are suggesting we add a chgrp -r gegroup /gevol if we are doing a non-rpm upgrade and there is an existing /gevol directory?  If so, that would be easy enough to add...

Reply:
I think that's what the code in `opengee-fusion/post-install.sh` does, e.g. when determining whether to pass the no-chown option to `geupgradeassetroot`.  The issue here is to determine when to do that ownership change.

Reply:
And now I do understand the concern completely, as being about when and why to set no-chown.